components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/PostAuthnMissingClaimHandler.java

```diff
@@ -167,6 +177,11 @@ private PostAuthnHandlerFlowStatus handlePostAuthenticationForMissingClaimsReque
                         context.getContextIdentifier());
                 uriBuilder.addParameter(FrameworkConstants.REQUEST_PARAM_SP,
                         context.getSequenceConfig().getApplicationConfig().getApplicationName());
+                if (context.getProperty(POST_AUTH_MISSING_CLAIMS_ERROR) != null) {
+                    uriBuilder.addParameter(POST_AUTH_MISSING_CLAIMS_ERROR,
```

Comment:
We can add a generic parameter for the URL.

Reply:
Fixed in 60f2f8e